Formula/nfdump.rb

```diff
@@ -11,8 +11,16 @@ class Nfdump < Formula
     sha256 "5bbd8ec9a51cde3c6ed0aaeb1b77a4f2ba59bea99d5304fa1415a7e2e0d5280b" => :mountain_lion
   end
 
+  option "with-readpcap", "Read netflow packets from a give pcap_file instead of the network. Is intended for debugging only."
```

Comment:
@apjanke, check the man page for nfcapd (not nfdump) which is included by this package.

@mikemcquaid, this is a debug only option, and I cannot imagine that the developers nor 99% of the users would want it included by default.